doc/guides/contributing/pull-requests.md

```diff
@@ -251,7 +251,9 @@ And on Windows:
 > vcbuild test
 ```
 
-(See the [running tests][] section of Building guide for more details.)
+For some configurations, running all tests might take a long time (an hour or
```

Comment:
I want to keep some kind of caveat like this ("For some configurations") because I don't believe the statement is true on fairly typical macOS and Linux systems.